### PR TITLE
Make commercial cards consistent

### PR DESCRIPTION
**Pull request by kelvin-chappell**
So that cards in Capi merchandising components are the same as cards in paid containers.

/cc @regiskuckaertz

### CODE REVIEW

File: commercial/app/views/contentapi/itemV2.scala.html [hunk 1]
```diff
@@ -30,7 +31,7 @@
 }{
 
     <div class="adverts__row">
-        @itemLargeCard(item, clickMacro)
+        @itemLargeCard(card, omnitureId)
```
Comment (regiskuckaertz) [inline commercial/app/views/contentapi/itemV2.scala.html]: so no more `clickMacro`?

**Comment by regiskuckaertz** [comment]
Thanks, that's awesome! :+1:

File: commercial/app/views/contentapi/itemV2.scala.html (hunk 1 above)
Comment (kelvin-chappell) [inline commercial/app/views/contentapi/itemV2.scala.html]: It gets passed in as part of the `targetUrl` of the card content.
That's built here: https://github.com/guardian/frontend/blob/kc-capi-card/common/app/common/commercial/CardContent.scala#L79

Comment (regiskuckaertz) [inline commercial/app/views/contentapi/itemV2.scala.html]: :heart_eyes:

**Pull request merged**